agenda-connectors-webapp/src/main/webapp/vue-app/agenda-connectors/exchange-connector/agendaExchangeConnector.js

```diff
@@ -52,14 +52,28 @@ export default {
     });
   },
 
-  getEvents(periodStartDate, periodEndDate){
+  getEvents(periodStartDate, periodEndDate) {
     return agendaExchangeService.getExchangeEvents(periodStartDate, periodEndDate)
       .then(events => {
         events.forEach(event => {
+          event.id = event.remoteId;
           event.type = 'remoteEvent';
           event.color = '#FFFFFF';
         });
         return events;
       });
+  },
+  pushEvent(event) {
+    const exchangeEvent = {
+      id: event.id,
+      summary: event.summary,
+      start: event.start,
+      end: event.end,
+      remoteProviderName: this.name,
+      remoteProviderId: 1,
```

Comment:
Normally this should be retrieved from backend as AGENDA_PROVIDER_ID field of EXO_AGENDA_REMOTE_PROVIDER table based on remoteProviderName as NAME field of the same table. The remoteProviderId should be used to store the remote event in EXO_AGENDA_REMOTE_EVENT table.